src/bindings/html/dom.js

```diff
@@ -81,19 +81,34 @@ export function translateMutations(view, langs, mutations) {
             targets.add(addedNode);
           }
         }
+        break;
     }
   }
 
-  targets.forEach(
-    target => target.childElementCount ?
-      translateFragment(view, langs, target) :
-      translateElement(view, langs, target));
+  if (targets.size === 0) {
+    return;
+  }
+
+  let elements = [];
+
+  targets.forEach(target => {
```

Comment:
Yeah, I know. Still, there's something about syntax like https://github.com/l20n/l20n.js/blob/cbd9c5162833fe5d84e4224ea28b4f016d231760/src/bindings/html/dom.js#L63-L67 which I've yet to get used to.